### ISSUE TITLE
Regarding career choices

### ISSUE THREAD

**Issue by akshansh2000** [tanaypratap/ama#217]
Hi Tanay!
I am currently a sophomore, pursuing B.Tech. in Information Technology.

You have always said one thing, _"Learn something which will get you money, and then focus on other things."_

Following this thought, I started out with _Web Development using JavaScript_ but didn't find it interesting, and got bored. I switched over to _Django_, the same story. Basically, I __did not like Web Development__.

I thought of trying something new, something that has enough jobs so as to get me some money, and tried out _Android Development_. Again, I could not bear with _Java_, nor _React Native_.

I tried _Machine Learning_ and _Natural Language Processing_, the same story.

I started using _Flutter_ for Android/iOS Development, and I am simply __in love with Flutter__. I thought that I now know what I have to do, but this directly contradicts the above statement of learning something that would get me money. _Flutter_ jobs haven't opened up yet and will take a couple of years before people start adapting to it.

I also like _AR/VR Development_, but this industry is the same as _Flutter_ i.e. not enough jobs, not enough adaption of technology.

Basically, I tried out many things which are supposed to get me a job in the next couple of years but didn't find any of them interesting. The ones I liked are all which won't help me start my career due to lack of opportunities/jobs.

What should I do in such a case? Should I work on something which I don't like just because I get money from it? Should I leave my interests (_Flutter_ and _AR/VR_) aside for some time to focus on other things and get back to them when I have job security, or just work so hard with them so as to be able to land a job in them without having any experience?

Thank You!

**Comment by tanaypratap** [comment]
What's it that you don't like about React but like about Flutter?

**Comment by akshansh2000** [comment]
Hey @tanaypratap!

I didn't particularly like React's package management. It breaks so often, and is harder to configure (than Flutter) if you're working on different machines.

Moreover, I did not like the JSX syntax, and I prefer Dart's programmatic approach a lot more.

The only benefit (that I found) in using React was really small app sizes.

**Comment by tanaypratap** [comment]
If you choose a language/framework based on ease of use (personal preference) than market demand then you'll be less sellable.
For freelancing, do whatever you wanna do. But if you need a job, you need to know what the job asks. 

Anyway, I think if you're programming in any language it's fine. But do cover the basics and be open to learn or work on anything.
